libs/blockscout-chains/src/lib.rs

```diff
@@ -0,0 +1,58 @@
+use reqwest_middleware::ClientBuilder;
+use reqwest_retry::{policies::ExponentialBackoff, RetryTransientMiddleware};
+use serde::Deserialize;
+use std::collections::HashMap;
+
+const CHAINS_URL: &str = "https://chains.blockscout.com/api/chains";
+
+pub async fn get_blockscout_chains() -> anyhow::Result<BlockscoutChains> {
+    let max_retries = 3;
+    let retry_policy = ExponentialBackoff::builder().build_with_max_retries(max_retries);
+    let client = ClientBuilder::new(reqwest::Client::new())
+        .with(RetryTransientMiddleware::new_with_policy(retry_policy))
+        .build();
+    let res = client.get(CHAINS_URL).send().await?;
+    let chains: BlockscoutChains = res.json().await?;
+    Ok(chains)
+}
+
+pub type BlockscoutChains = HashMap<i64, BlockscoutChainData>;
+
+#[derive(Deserialize, Debug)]
+#[serde(rename_all = "camelCase")]
+pub struct BlockscoutChainData {
+    pub name: String,
+    pub description: String,
+    pub ecosystem: Ecosystem,
+    pub is_testnet: Option<bool>,
```

Comment:
Nit: just interested, what is the case when `is_testnet` is missing?

Reply:
Just copied that from the original interface:
https://github.com/blockscout/chainscout/blob/7657da41a7dda8c180cfb9ee2fc858c3d27c2dfc/types/index.ts#L12

Reply:
Oh, I see. May we also ask Max why it was the case?

Reply:
Decided to leave the `Option` type as was discussed on the call